cypress/integration/list.js

```diff
@@ -171,16 +171,50 @@ describe('List Page', () => {
 
     describe('rowClick', () => {
         it('should accept a function', () => {
-            cy.contains('Fusce massa lorem, pulvinar a posuere ut, accumsan ac nisi').parents('tr').click();
+            cy.contains(
+                'Fusce massa lorem, pulvinar a posuere ut, accumsan ac nisi'
+            )
+                .parents('tr')
+                .click();
             cy.contains('Summary').should(el => expect(el).to.exist);
         });
 
         it('should accept a function returning a promise', () => {
             LoginPage.navigate();
             LoginPage.login('user', 'password');
             ListPageUsers.navigate();
-            cy.contains('Annamarie Mayer').parents('tr').click();
+            cy.contains('Annamarie Mayer')
+                .parents('tr')
+                .click();
             cy.contains('Summary').should(el => expect(el).to.exist);
         });
     });
+
+    describe('expand panel', () => {
+        it('should show an expand button opening the expand element', () => {
+            cy.contains('1-10 of 13'); // wait for data
+            cy.get('[role="expand"]')
+                .eq(0)
+                .click();
+            cy.get('[role="expand-content"]').should(
```

Comment:
I avoid magic strings expectations as they prevent tools such as the IDE autocompletion and type checking to do their job. I suggest to use the following form: `.should(el => expect(el).to.contain(''))`